public/app/plugins/datasource/grafana-azure-monitor-datasource/components/SubscriptionField.tsx

```diff
@@ -95,9 +97,33 @@ const SubscriptionField: React.FC<SubscriptionFieldProps> = ({
     [query, onQueryChange]
   );
 
+  const onSubscriptionsChange = useCallback(
+    (change: Array<SelectableValue<string>>) => {
+      if (!change) {
+        return;
+      }
+
+      query.subscriptions = change.map((c) => c.value ?? '');
+
+      onQueryChange(query);
+    },
+    [query, onQueryChange]
+  );
+
   const options = useMemo(() => [...subscriptions, variableOptionGroup], [subscriptions, variableOptionGroup]);
 
-  return (
+  return multiSelect ? (
+    <Field label="Subscriptions">
+      <MultiSelect
+        isClearable
+        value={findOptions(subscriptions, query.subscriptions)}
+        inputId="azure-monitor-subscriptions-field"
+        onChange={onSubscriptionsChange}
+        options={options}
+        width={40}
```

Comment:
Can we use the same width for this as for the service picker? Think it's 38

Reply:
Is there a way the sub picker adjust its width according to the selected values while maintaining a min width?